modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/consensus/ConsensusManager.scala

```diff
@@ -52,15 +53,23 @@ object ConsensusManager {
   ): F[ConsensusManager[F, Key, Artifact]] = {
     val logger = Slf4jLogger.getLoggerFromClass[F](ConsensusManager.getClass)
 
+    def exchangeRegistration(peerId: PeerId, peer: Peer): F[Unit] =
```

Comment:
I know I can, but for some reason we didn't do it in one of the places where this function is now used. We have `peerId` and `peer` separated there.

Reply:
Private quantifier is not allowed in this place. It's a def inside of another def - can't be accessed from the outside.

Reply:
I don't want to check the root of why we pass these both separate. If you say they are the same I will gladly change it.

Reply:
We pass both because when `Some(Peer)` gets removed from cluster it is denoted as `None` and we want to have information about which `PeerId` got removed, hence passing `(PeerId, Option[Peer])`.